libs/boost/Makefile

```diff
@@ -421,7 +440,7 @@ endef
 
 define BuildBoostLibrary
   define Package/boost-$(1)/install
-	$(call Package/boost/Default/install,$$(1),$(1))
+    $(call Package/boost/Default/install,$$(1),$(1))
```

Comment:
I think the indentation here needs to be a tab instead of spaces since it's for recipes